### PR TITLE
Update proxyUrls returned for Cromwell App to link to Cromwell Swagger page [BW-869]

### PR DESCRIPTION
**Pull request by cjllanwarne**
This PR updates the response from `getApp` to reference the correct path to the Cromwell swagger page, with correct parameters to set up the swagger service with the correct yaml.

### CODE REVIEW

**Review by kyuksel — APPROVED** [review]
LGTM 👍🏼 
Haven't tested it.

**Review by Qi77Qi — COMMENTED** [review]
For other cases, the proxy url is where the app can be accessed(think UI uses this for serving the page). Does Cromwell have a UI similar to that? Or users are supposed access via swagger?

File: core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala [hunk 1]
```diff
@@ -357,8 +359,14 @@ final case class App(id: AppId,
                      extraArgs: List[String]) {
   def getProxyUrls(project: GoogleProject, proxyUrlBase: String): Map[ServiceName, URL] =
     appResources.services.map { service =>
-      (service.config.name,
-       new URL(s"${proxyUrlBase}google/v1/apps/${project.value}/${appName.value}/${service.config.name.value}"))
+      val proxyPath = s"google/v1/apps/${project.value}/${appName.value}/${service.config.name.value}"
+      appType match {
```
Comment (rtitle) [inline core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala:363]: I guess we kind of made the assumption previously that `service.name` (e.g. "cromwell-service", defined [here](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/reference.conf#L420)) is the leaf element of the proxy URL. But this is not the case for accessing Swagger, I guess.

I wonder if this is a one-off for Cromwell or if other custom apps will have this issue. If it's common, we might want to make `proxyExtraPath` or something configurable in the app definition.

**Review by rtitle — APPROVED** [review]
This seems fine for now… also echoing Qi whether eventually Cromwell-as-an-app will have a different entry point rather than its swagger page, in which case this will need to be changed.

**Comment by cjllanwarne** [comment]
jenkins retest

**Comment by cjllanwarne** [comment]
jenkins retest

**Review by aednichols — APPROVED** [review]

**Pull request merged**